administrators.rst

```diff
@@ -680,16 +681,19 @@ Fields:
 * **Path**: *Not required: see below*.
 * **Staging path**: the absolute path to a staging area on the same server as
   the storage service.
+* **Bucket**: the name of the bucket to use. *Not required: see below*
 * **S3 Endpoint URL**: the URL of the S3 endpoint, e.g.
   ``https://s3.amazonaws.com``.
-* **Access Key ID to authenticate**: the public key generated by S3.
-* **Secret Access Key to authenticate with**: the secret key generated by S3.
+* **Access Key ID to authenticate**: the public key generated by S3. *Not required: see below*
```

Comment:
We follow an 80-character line limit convention; this line and the next just need to be truncated to fit that. Because this is in a list, the second line needs to be indented by two spaces (to line up with the double asterisk.)